tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -1271,6 +1285,14 @@ Given('user {string} from server {string} has accepted the last pending share',
   }
 })
 
+When('user {string} from server {string} accepts the last pending share', function(user, server) {
+  if (server === backendHelper.BACKENDS.remote) {
+    return backendHelper.runOnRemoteBackend(() => sharingHelper.acceptLastPendingShare(user))
+  } else {
+    return sharingHelper.acceptLastPendingShare(user)
+  }
+})
+
```

Comment:
why adding a new definition when we already have one??

Reply:
My mistake. I couldn't find the step definition the first time I searched it and created new. I have removed it in my latest commit.